src/app/space/create/codebases/services/codebases.service.ts

```diff
@@ -110,6 +112,24 @@ export class CodebasesService {
       });
   }
 
+  /**
+   * Delete codebase
+   *
+   * @param {Codebase} codebase codebase to delete
+   * @returns {Observable<Codebase>}
+   */
+  deleteCodebase(codebase: Codebase): Observable<Codebase> {
```

Comment:
Please add a unit test to the service.spec.ts.

Reply:
done https://github.com/fabric8-ui/fabric8-ui/pull/2236/files#diff-6037b30f05a1462d22651009442ba9a8R251